compiler/damlc/tests/src/platform-independence-test.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+
+DAR1=$(sha1sum $1)
+DAR2=$(sha1sum $2)
+
+if [ $DAR1 -eq $DAR2 ]
+then
+  exit 0
+else
+  exit 1
+fi
```

Comment:
This should be equivalent to just the condition, i.e.
```suggestion
[ $DAR1 -eq $DAR2 ]
```

Equivalently, the whole script here could be replaced by a call to `diff`.